### PR TITLE
feat(python): add `replace_all_objects`

### PR DESCRIPTION
**Pull request by shortcuts**
## 🧭 What and Why

🎟 JIRA Ticket: https://algolia.atlassian.net/browse/DI-1774

### Changes included:

add the `replace_all_objects` helper, pretty much the same as the one in today's client, just for python3 and adapted to use our clients.

Tested via the playground, works fine

### CODE REVIEW

File: playground/python/app/search.py
Comment (shortcuts) [inline playground/python/app/search.py:1]: this is here for you to test

File: templates/python/search_helpers.mustache [hunk 1]
```diff
@@ -198,4 +198,63 @@
...
+
+    async def replace_all_objects(self, index_name: str, objects: Union[List[Dict[str, Any]], Iterator[Dict[str, Any]]], safe: Optional[bool] = False, request_options: Optional[Union[dict, RequestOptions]] = None) -> List[ApiResponse[str]]:
+        """
+        Helper: Replaces all objects (records) in the given `index_name` with the given `objects`. A temporary index is created during this process in order to backup your data.
+        """
+        tmp_index_name = self.create_temporary_name(index_name)
+        responses = List[ApiResponse[str]]()
+        copy_resp = await self.operation_index(
+                index_name=tmp_index_name,
+                operation_index_params={
```
Comment (millotp) [inline templates/python/search_helpers.mustache:217]: are you not supposed to use the model instead of a dict ?

**Review by millotp — COMMENTED** [review]
looks good !

File: templates/python/search_helpers.mustache (hunk 1 above)
Comment (shortcuts) [reply templates/python/search_helpers.mustache:217]: I can, both works!

Comment (millotp) [reply templates/python/search_helpers.mustache:217]: maybe we should stick to the models to have validation

Comment (shortcuts) [reply templates/python/search_helpers.mustache:217]: yup make sense

**Review by Fluf22 — APPROVED** [review]
🆗

**Review by Fluf22 — APPROVED** [review]
Let's go for real this time

**Pull request merged**